### PR TITLE
Laravel 8 support

### PR DESCRIPTION
**Pull request by UpwardProcess**
Laravel 8 support.

### CODE REVIEW

**Comment by UpwardProcess** [comment]
Pinging @andywer (based on your own comment on a previous PR).

File: composer.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
     },
     "extra": {
         "component": "package",
-        "frameworks": ["Laravel 7"]
+        "frameworks": ["Laravel 8"]
```
Comment (andywer) [inline composer.json]: If it works with Laravel 8 out-of-the-box and still works with Laravel 7, then let's make it `"frameworks": ["Laravel 7", "Laravel 8"]`, shall we? 🙂

**Comment by andywer** [comment]
Thanks for the PR, @UpwardProcess!

I left one question for you.

File: composer.json (hunk 1 above)
Comment (UpwardProcess) [reply composer.json]: Hi @andywer, updated in the latest commit.

**Pull request merged**